website/en/docs/lang/subtypes.md

```diff
@@ -136,3 +136,37 @@ type Func2 = (number, string) => void;
 let func1: Func1 = (a: number) => {};
 let func2: Func2 = func1;
 ```
+
+#### Subtypes of unions of functions <a class="toc" id="toc-subtypes-of-unions-of-functions" href="#toc-subtypes-of-unions-of-functions"></a>
+
+A function is a subtype of another when the input is contravariant (_less specific type / supertype_) and the output is covariant (_more specific type / subtype_).
```

Comment:
The wording here makes me imagine that some functions don't have contravariant inputs and covariant outputs, but all functions have this behavior.

It's more precise to say that function types are contravariant with respect to their parameter types and covariant with respect to their return type. That is, contra- and covariance are properties of the function type.

However, I think what you have is easier to understand. I would just remove the variance stuff (at least to start with):

"A function is a subtype of another when its argument types are less specific (supertype) or its return type is more specific (subtype)."

You can then go on to introduce the technical jargon: "In technical terms, we say that function types are contravariant with respect to their parameter types and covariant with respect to their return type."